Documentation/components/alerts.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/components/alerts.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dexterous UI- alerts</title>
+  </head>
+  <body>
+    <div class="parent-container">
+      <h3>Alert</h3>
+      <p>
+        An alert displays a short, important message in a way that attracts the
+        user's attention without interrupting the user's task.
+      </p>
+
+      <h3>Simple Alert</h3>
```

Comment:
Use h4. Alerts is your main heading. Simple Alerts is a sub-heading. Maintain visual hierarchy.